src/iframe/src/Iframe.js

```diff
@@ -933,12 +944,16 @@ class Iframe extends Component {
               restart
             </button>
 
-            <div
-              className={classNames('fps', {
-                hide: isPaused
-              })}
+            <div className={classNames('stat-values', {
+              hide: isPaused
+            })}
             >
-              fps (avg): <span>{fps}</span>
+              <div className={classNames('stat')}>
+                mouse: (<span>{this.state.mousePosition.x}</span>, <span>{this.state.mousePosition.y}</span>)
```

Comment:
I understand avoiding visually cluttering the display. But... could we have this in `input` ? :-)
A number of genres benefit from pointer input (lemmings, duck hunt, simcity...)

Reply:
Let me think about it.